config.py

```diff
@@ -16,3 +15,39 @@
 # This dict holds aspect ratio data for hydrograph rendered in static/js/hydrograph.js. Keys are 'height' and 'width'
 HYDRO_META = {}
 
+# Map configuration
+
+SPATIAL_REFERENCE_ENDPOINT = 'http://spatialreference.org/ref/epsg/${epsg_code}/proj4/'
+PROJECTION_EPSG_CODE = '4326'
+
+#
+BOUNDING_BOX = [-94.294625, 41.015733, -91.424186, 43.102420]
+
+BACKGROUND_FILE = 'floodviz/static/geojson/states.json'
+REFERENCE_FILE = 'floodviz/static/geojson/reference.json'
+# this contains data that needs no further transformation before being sent to the javascript
+MAP_CONFIG = {
```

Comment:
I would recommend that the map_config include the width, height, and scale. If you want the bounding box represented as a geojson, then do so in the view or in the application initialization.